packages/flutter/test/services/system_chrome_test.dart

```diff
@@ -9,6 +9,8 @@ import 'package:flutter_test/flutter_test.dart';
 
 void main() {
   testWidgets('SystemChrome overlay style test', (WidgetTester tester) async {
+    await tester.idle(); // Flush any outstanding microtasks.
```

Comment:
there shouldn't be any at this point in the test.

Reply:
did you determine why this is required?

Reply:
Actually, I determined that it wasn't, but I forgot to revert it.  Reverted.